app/javascript/components/mapping-to-domains/TermCard.jsx

```diff
@@ -107,7 +109,10 @@ export default class TermCard extends Component {
             (disableClick ? "" : selected ? "" : " cursor-pointer")
```

Comment:
Again, better avoid nested ternary operators as they're not always easy to read. Here, a single expression can be used: `disableClick || selected ? "" : " cursor-pointer"`.